### PR TITLE
pdal: 2.4.0 (new-package)

### PR DESCRIPTION
**Pull request by MehdiChinoune**
I couldn't package 2.3.0 because it fails to build.
I will update it to the final release once It's released, but It is needed to build VTK 9.1

### CODE REVIEW

File: mingw-w64-pdal/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  ${MINGW_PREFIX}/bin/cmake \
+    -Wno-dev \
+    -G"Ninja" \
+    ${extra_config[@]} \
+    -DCMAKE_INSTALL_PREFIX="${MINGW_PREFIX}" \
+    -DBUILD_SHARED_LIBS=ON \
+    -DWITH_TESTS=OFF \
+    ../PDAL-${pkgver/RC1}-src
+
+  ${MINGW_PREFIX}/bin/cmake --build . || ${MINGW_PREFIX}/bin/cmake --build . || ${MINGW_PREFIX}/bin/cmake --build .
```
Comment (lazka) [inline mingw-w64-pdal/PKGBUILD:42]: build problems?

Comment (MehdiChinoune) [reply mingw-w64-pdal/PKGBUILD:42]: Sorry!

**Comment by MehdiChinoune** [comment]
2.4.0 was released!

**Comment by MehdiChinoune** [comment]
ready

**Pull request merged**